keps/sig-multicluster/1645-multi-cluster-services-api/README.md

```diff
@@ -655,6 +659,8 @@ metadata:
 spec:
   ips:
   - 42.42.42.42
+  ipFamilies:
+    - IPv4
```

Comment:
AFAIK, IPV4 and IPV6 format are quite different, are the formats in the "ips" field alone not enough for the consumer to discern? I assume that the consumer knows what IP family it can support.

Reply:
This field is more aimed toward the allocation of the `IPs` than what happens when/before  the `IPs` are actually allocated a bit like the current type field actually or even how the `ipFamilies` field behave on a regular Service. For Cilium we would most likely want to do an intersection of all the `ipFamilies` in the exported Services which would put this as relevant as the other fields essentially.